les/client.go

```diff
@@ -234,6 +234,11 @@ func (s *LightEthereum) APIs() []rpc.API {
 			Version:   "1.0",
 			Service:   NewLightClientAPI(s),
 			Public:    true,
+		}, {
+			Namespace: "les",
+			Version:   "1.0",
+			Service:   NewPrivateLightClientAPI(s),
```

Comment:
Instead of making a private light client API seperate from a "public" version, Id recommend merging the new function into `LightClientAPI` and making that API private. As far are I can tell, there is no reason to have a "public" light client API.